input/armor-acquisition/blacklionchest.json

```diff
@@ -5,11 +5,13 @@
     8115,8121,8122,8617,8618,8621,
     // Seven Reapers
     9376,9384,9447,9890,9891,9894,10098,10100,10106,10124,10129,10131,11059,11068,11080,11110,11113,11134,
-    // Standalone - Head Armor
-    7225,7233,7241,7782,7800,7804,9789,9790,9792,10847,10849,10867,
-    // Standalone - Shoulder Armor
+    // Standalone: Other - Head
```

Comment:
Yeah, my bad, I was uploading a few cleaned files but then it conflicted and it turned into a mess, woops.

Those comments are a 1=1 replica of the categories from the group files, thinking about the future more than anything else, so when you put them in the group under a section then you go to the filters and put them under the same section.

You could find them easily by just looking for that section in the live website itself, but yeah, not quite clear.

Ideally you'd look into updating the wiki here and there to retrieve that data right from there, but you can't really rely on it being up to date either, so I'd stick with having extra files around. It's kinda sad because it would be great if the site retrieved gem store items with their normal prices, but oh well.

Reply:
Well, there's a disabled ```v2/gemstore/catalog``` api, so perhaps one day data will be there - although presumably it will be current rather than historical gemstore items.